packages/tools/src/tools/annotation/AngleTool.ts

```diff
@@ -69,6 +70,8 @@ class AngleTool extends AnnotationTool {
       supportedInteractionTypes: ['Mouse', 'Touch'],
       configuration: {
         shadow: true,
+        showAngleArc: true,
```

Comment:
can we make it off by default, since it is not a major release, i know people have tests and those would fail immediately

Reply:
i missed to put it false after the test, i did add a `angleArcLineDash` style for the arc

Reply:
it's ready for review again